distribution/src/bin-filemode-755/hz-healthcheck.sh

```diff
@@ -8,7 +8,7 @@ print_help() {
     echo "  -d, --debug       : Prints curl error output."
     echo "HTTPs related (TLS enabled):"
     echo "      --https       : Uses HTTPs protocol for REST calls. (no parameter value expected)"
-    echo "      --cacert      : Defines trusted PEM-encoded certificate file path. It's used to verify member certificates."
+    echo "      --ca-cert     : Defines trusted PEM-encoded certificate file path. It's used to verify member certificates."
```

Comment:
This option is now different than the one in hz-cluster-admin.

Reply:
Fixed.

Reply:
Also, remove .sh from this file name